README.md

```diff
@@ -114,7 +114,7 @@ Read our [contribution guidelines](CONTRIBUTING.md) on how to submit issues and
 ### Dependencies
 
 The Nakadi server is a Java 8 [Spring Boot](https://projects.spring.io/spring-boot/) application. 
-It uses [Kafka 1.1.1](https://kafka.apache.org/11/documentation.html) as its broker and
```

Comment:
I am fine with upgrading to 2.0.0, but our current deployment uses 1.1.1. It would be great if you could rephrase this sentence, to indicate that Kafka 1.0.0 and up are supported. At the moment, it is entirely possible for people running 1.0.0+ versions of Kafka to run Nakadi.

Reply:
Indeed

Reply:
Hi @cbornet , are you still working on this?

Reply:
Sorry I forgot about it. I'll do the changes.

Reply:
done